common/src/main/java/com/microsoft/identity/common/adal/internal/net/DefaultConnectionService.java

```diff
@@ -52,6 +60,48 @@
      */
     public DefaultConnectionService(Context ctx) {
         mConnectionContext = ctx;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            this.registerNetworkCallback();
```

Comment:
```suggestion
            registerNetworkCallback();
```

(We generally aren't using `this` in our code unless some variable shadowing is happening)